service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/OrdersNeededController.java

```diff
@@ -73,6 +109,20 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
             data.put("otherOrderType", "NO");
         }
 
+        Court selectedCourt = getCourtSelection(courtID);
+
+        if (Objects.nonNull(selectedCourt)) {
+            data.put("court", selectedCourt);
+        }
+
+        if (ordersFieldName.equals("ordersSolicitor")) {
```

Comment:
may be misremembering but can we do this anymore - was the set of values for `orders` diverging from `ordersSolicitor`

Reply:
For ticket DFPL-670, I'm going to approach the problem another way after speaking to Sara Parry a bit about the requirements so leaving this like this should be fine (for now at least)